crates/experimental-api/tests/common/mod.rs

```diff
@@ -14,6 +17,24 @@ impl From<iamgroot::jsonrpc::Error> for Error {
     }
 }
 
+impl From<std::env::VarError> for Error {
+    fn from(error: std::env::VarError) -> Self {
+        Self::Var(error)
+    }
+}
+
+impl From<beerus_experimental_api::exe::err::Error> for Error {
+    fn from(error: beerus_experimental_api::exe::err::Error) -> Self {
+        Self::Exe(error)
+    }
+}
+
+impl From<serde_json::Error> for Error {
+    fn from(error: serde_json::Error) -> Self {
+        Self::Json(error)
+    }
+}
```

Comment:
Since we have `thiserror` in this crate, all this code could be reduced to `#[from]` annotations.

Reply:
Done.